go.mod

```diff
@@ -80,3 +80,5 @@ replace gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.2.2
 replace honnef.co/go/tools => honnef.co/go/tools v0.0.1-2019.2.3
 
 replace sigs.k8s.io/yaml => sigs.k8s.io/yaml v1.1.0
+
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.6
```

Comment:
Move this to the larger replace block since it's not likely to be temporary.

Reply:
Done.